homeassistant/components/reolink/switch.py

```diff
@@ -108,6 +108,14 @@ class ReolinkNVRSwitchEntityDescription(
         value=lambda api, ch: api.ptz_guard_enabled(ch),
         method=lambda api, ch, value: api.set_ptz_guard(ch, enable=value),
     ),
+    ReolinkSwitchEntityDescription(
+        key="ptz_patrol",
+        translation_key="ptz_patrol",
+        icon="mdi:map-marker-path",
+        supported=lambda api, ch: api.supported(ch, "ptz_patrol"),
```

Comment:
Consider adding `entity_category=EntityCategory.CONFIG,`

Reply:
I was thinking about this, but hitting on/off will directly start/stop the movement off the camera.
So I thougth it was more of a control than a setting.
But this is debatable since some people may leave the patrol mode constantly on to have the camera always moving in a pettern indefenitely.

The other PTZ functions are controls, like move up/down/left/right/stop.

I lean towards keeping it a control and thus not adding the `EntityCategory.CONFIG`.
@Kane610 what do you think?
If you think it is better as a config I will change it.

Reply:
I typically use config in all the cases that is not the device primary function. That's why I added the comment. Your reasoning is sound I have nothing more to add :)